machine/machine_core/machine.py

```diff
@@ -298,7 +303,7 @@ def stop_cockpit(self):
         """Stop Cockpit.
         """
         if self.ostree_image:
-            self.execute("podman ps --quiet --all --filter ancestor=cockpit/ws | xargs --no-run-if-empty podman rm -f")
+            self.execute("echo {0} | xargs --no-run-if-empty podman rm -f".format(self.get_cockpit_container()))
```

Comment:
Likewise, ther can only ever be one container with a given  name, so a simple `f"podman rm -f {self.get_cockpit_container()}"` would do.

Reply:
tests did not succeed - reposting with suggestions

Reply:
I think the xargs is needed because sometimes podman ps command returns an empty list.

Reply:
Of course you are right.